stable/redis-ha/Chart.yaml

```diff
@@ -5,7 +5,7 @@ keywords:
 - redis
 - keyvalue
 - database
-version: 3.1.3
+version: 3.1.4
```

Comment:
Could you add the url `https://github.com/oliver006/redis_exporter` to the sources list as well?

Reply:
Done